### PR TITLE
invert expect_crash_on_failure config option defaults

### PR DESCRIPTION
**Pull request by bmc-msft**
This matches the existing job template behavior

### CODE REVIEW

**Review by ranweiler — APPROVED** [review]

File: src/agent/onefuzz-agent/src/local/libfuzzer_fuzz.rs [hunk 1]
```diff
@@ -94,9 +94,9 @@ pub fn build_shared_args() -> Vec<Arg<'static, 'static>> {
         Arg::with_name(CHECK_FUZZER_HELP)
             .takes_value(false)
             .long(CHECK_FUZZER_HELP),
-        Arg::with_name(DISABLE_EXPECT_CRASH_ON_FAILURE)
+        Arg::with_name(EXPECT_CRASH_ON_FAILURE)
             .takes_value(false)
```
Comment (ranweiler) [inline src/agent/onefuzz-agent/src/local/libfuzzer_fuzz.rs:98]: Our now naming now agrees with the non-local `libfuzzer-fuzz` task, but there the `expect_crash_on_failure` defaults defaults to `true`. Do we want to do that here, too, to invert the name, but keep the semantics?

Comment (ranweiler) [reply src/agent/onefuzz-agent/src/local/libfuzzer_fuzz.rs:98]: Ref: https://github.com/microsoft/onefuzz/blob/458a142232a288950793d52426403d9ef5cebe13/src/agent/onefuzz-agent/src/tasks/fuzz/libfuzzer_fuzz.rs#L64-L65

Comment (bmc-msft) [reply src/agent/onefuzz-agent/src/local/libfuzzer_fuzz.rs:98]: good point.  I added an additional helper for the default value to be false.

File: src/agent/onefuzz-agent/src/tasks/fuzz/libfuzzer_fuzz.rs [hunk 2]
```diff
@@ -61,7 +65,7 @@ pub struct Config {
     #[serde(default = "default_bool_true")]
     pub check_fuzzer_help: bool,
 
-    #[serde(default = "default_bool_true")]
+    #[serde(default = "default_bool_false")]
```
Comment (ranweiler) [inline src/agent/onefuzz-agent/src/tasks/fuzz/libfuzzer_fuzz.rs:68]: If you like, this could be:
```suggestion
    #[serde(default)]
```

**Review by ranweiler — APPROVED** [review]

**Pull request merged**